apps/script/checkEntryPoints.js

```diff
@@ -84,7 +78,7 @@ const SILENCED = [
   'netsim',
   'pd',
   'plc',
-  'publicKeyCryptography',
+  'projectsList',
```

Comment:
From the comments above:

  * You may be tempted to add your entry point to the list of silenced entry  | |
  * | |  points, but that would be a mistake: